daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -180,7 +180,7 @@ Com_CharIsOneOfCharset
 */
 static bool Com_CharIsOneOfCharset( char c, const char *set )
 {
-	for (unsigned i = 0; i < strlen( set ); i++ )
+	for (std::size_t i = 0, len = strlen( set ); i < len; i++ )
```

Comment:
Fixed. 
I prefer reduce scope of vars to least possible (less probability of unneeded intersection with out side vars, reduce chance of incorrect usage).